packages/components/src/components/Popper.vue

```diff
@@ -1,9 +1,11 @@
 <template>
-  <div class="popper" @mouseover="hover = true" @mouseleave="hover = false">
+  <div class="popper">
```

Comment:
This breaks the value hover on nodes in the trace view. Perhaps we need a prop that controls whether or not the popper becomes active on hover or on click?